README.md

```diff
@@ -319,7 +331,7 @@ use of version properties for these versions available in the YAKS base image:
 You can add dependencies also by specifying the dependencies as command line parameter when running the test via `yaks` CLI.
 
 ```bash
-yaks test --dependencies org.apache.camel:camel-groovy:@camel.version@ camel-route.feature
+yaks test --dependency org.apache.camel:camel-groovy:@camel.version@ camel-route.feature
```

Comment:
I think the option is still called `dependencies` you just have changed the type from `string` to `string[]`

Reply:
I've renamed it to `dependency` also in the code because I switched from the CSV model to a more standard way.. so now, if you want to declare multiple dependencies, you've to write: `yaks test --dependency org.my:one:vers --dependency org.my:two:vers`.

Reply:
Ah I just had a look at the options where it days `dependencies []string` but the option is renamed to `dependency` as you say. Just did not see that, thanks for clarification!